### PR TITLE
venue report form is done

### PR DESCRIPTION
**Pull request by multimediaschneider**
venueId is still hard coded in report-form.tsx

### CODE REVIEW

File: app/api/report-venue/route.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { NextResponse } from "next/server";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
```
Comment (louziferr) [inline app/api/report-venue/route.ts]: please import prisma from lib/db

File: components/report-form.tsx [hunk 2]
```diff
@@ -9,22 +12,53 @@ import {
...
-import { FormDescription } from "./ui/form";
 import { Textarea } from "./ui/textarea";
 
 export default function ReportForm() {
+  const [issue, setIssue] = useState("");
+  const [datetime, setDatetime] = useState<Date | null>(null);
+  const [detail, setDetail] = useState("");
+  const router = useRouter();
+
+  const venueId = "920521f5-37a2-46ac-9e03-304763998903";
```
Comment (louziferr) [inline components/report-form.tsx]: get a ID from the DB.

await prisma.venue.findUnique({
     where: {
    // venue.name == "Erich Zeigner Tischtennisplatte" }
})

Comment (multimediaschneider) [reply components/report-form.tsx]: seed script has to be initiated again for changes to apply

venues have a fixed id now to make it work on any machine

react-hook-form implemented

date-time-picker in one window now

date-time-picker automatically sets present day and time

File: app/api/venues/route.ts
Comment (dnmct) [inline app/api/venues/route.ts:1]: wofür wird diese route benutzt?

File: app/globals.css
Comment (dnmct) [inline app/globals.css:1]: bitte here noch die merge conflict marker rausnehmen

File: prisma/migrations/20240829084425_add_unique_const/migration.sql
Comment (dnmct) [inline prisma/migrations/20240829084425_add_unique_const/migration.sql:1]: migrations kannst du löschen wir benutzen push um die datenbank zu updaten

**Review by dnmct — CHANGES_REQUESTED** [review]

**Review by dnmct — APPROVED** [review]

**Pull request merged**